HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -348,57 +348,57 @@ def self.run_validator(hpxml_doc)
       '/HPXML/Building/BuildingDetails/Systems/HVAC/HVACPlant/HeatingSystem[HeatingSystemType/Furnace]' => {
         '../../HVACDistribution[DistributionSystemType/AirDistribution | DistributionSystemType[Other="DSE"]]' => one_or_more, # See [HVACDistribution]
         'DistributionSystem' => one,
-        'HeatingSystemFuel[text()="natural gas" or text()="fuel oil" or text()="fuel oil 1" or text()="fuel oil 2" or text()="fuel oil 4" or text()="fuel oil 5/6" or text()="diesel" or text()="propane" or text()="kerosene" or text()="electricity" or text()="wood" or text()="wood pellets"]' => one,
+        'HeatingSystemFuel[text()="natural gas" or text()="fuel oil" or text()="fuel oil 1" or text()="fuel oil 2" or text()="fuel oil 4" or text()="fuel oil 5 or 6" or text()="diesel" or text()="propane" or text()="kerosene" or text()="electricity" or text()="wood" or text()="wood pellets"]' => one,
```

Comment:
Not sure why you're wanting to change all of these, but it's not correct. Refer to the [HXPML enumeration](https://hpxml.nrel.gov/datadictionary/3.0.0/Building/BuildingDetails/Systems/HVAC/HVACPlant/HeatingSystem/HeatingSystemFuel)

Reply:
Hmm.  In the case where a child element is `foo/bar[text()=a or ...]`, I wanted to split the element by slash and choose only bar[text()=a or ...], but it didn't work when there is Fuel 5/6 and I ended up changing the name of this fuel type. I'll have to think of another way.